zebra/rib.h

```diff
@@ -371,7 +371,7 @@ extern void _route_entry_dump(const char *func, union prefixconstptr pp,
 			      union prefixconstptr src_pp,
 			      const struct route_entry *re);
 
-extern void rib_lookup_and_dump(struct prefix_ipv4 *p, vrf_id_t vrf_id);
```

Comment:
removing rib_lookup_and_dump is a separate commit that has nothing to do with this code.  Don't mind it being removed but let's keep commits to 1 thing so reviewers as well as future people have an easier time looking at this particular commit

Reply:
according to comment and actual usage - it was old function for rpm_read only, not used any more 
so I just reuse it, naming properly.

Of course I can just introduce another function keeping this one no-op (as it now), as you said.

Reply:
updated as proposed